### PR TITLE
Support ipv6 in ports

### LINKED ISSUE #1136: Support for ipv6 ports annotations
Please add IPv6 support for conversion from docker-compose to kubernetes

```
FATA 1 error(s) decoding:

* error decoding 'Ports': Invalid format to parse. fe80::6ec2:17ff:fe84:4270:8096:8096 should match template ip:hostPort:containerPort
```

Example ports service configuration:
```
    ports:
      - "192.168.1.41:8096:8096"
      - "fe80::6ec2:17ff:fe84:4270:8096:8096"
```

### PR DESCRIPTION
**Pull request by hangyan**
fix #1136

### CODE REVIEW

**Comment by Madic-** [comment]
Any news on the fix before it gets closed?

File: pkg/loader/compose/v1v2.go [hunk 1]
```diff
@@ -109,24 +109,26 @@ func loadPorts(composePorts []string) ([]kobject.Ports, error) {
...
 
-		if len(justPorts) == 3 {
+		if len(justPorts) >= 3 {
 			// ex. 127.0.0.1:80:80
+			// ex. fe80::6ec2:17ff:fe84:4270:8096:8096
 
+			l := len(justPorts)
 			// Get the IP address
-			hostIP := justPorts[0]
+			hostIP := protocolCheck[0][:len(protocolCheck[0])-2-(len(justPorts[l-2])+len(justPorts[l-1]))]
```
Comment (cdrage) [inline pkg/loader/compose/v1v2.go]: This seems a bit hacky IMO... 

Any way to use a library or a put this in a separate function to make sure we get just the IP address?

**Pull request closed without merging**